ts/screens/authentication/TemporaryBlockedCie.tsx

```diff
@@ -0,0 +1,108 @@
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import IconFont from "../../components/ui/IconFont";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+type Props = OwnProps;
+type State = Readonly<{
+  securityCode: Option<string>;
+  holder: Option<string>;
+}>;
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: customVariables.contentPadding
+  },
+  text: {
+    fontSize: customVariables.fontSizeBase
+  }
+});
+
+class TemporaryBlockedCie extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render(): React.ReactNode {
+    const cancelButtonProps = {
+      block: true,
+      cancel: true,
+      onPress: (): boolean => this.props.navigation.goBack(),
+      title: I18n.t("global.buttons.cancel")
+    };
+    const retryButtonProps = {
+      block: true,
+      primary: true,
+      onPress: () => undefined,
+      title: I18n.t("authentication.cie.temporaryBlockedCieDoneButton")
+    };
+
+    return (
+      <Container>
+        <BaseScreenComponent goBack={true}>
+          <View style={styles.contentContainerStyle}>
+            <H1>{I18n.t("authentication.cie.temporaryBlockedCieTitle")}</H1>
+            <View spacer={true} />
+            <Text style={styles.text}>
+              {I18n.t("authentication.cie.temporaryBlockedCieContent1")}
+            </Text>
+            <Text style={styles.text}>
+              {I18n.t("authentication.cie.temporaryBlockedCieContent2")}
+              <Text style={styles.text} bold={true}>
+                {I18n.t("authentication.cie.temporaryBlockedCieContent2-bold")}
+              </Text>
+              <Text style={styles.text}>
+                {I18n.t("authentication.cie.temporaryBlockedCieContent3")}
+              </Text>
+            </Text>
+            <Text style={styles.text}>
+              {I18n.t("authentication.cie.temporaryBlockedCieContent4")}
+              <Text style={styles.text} bold={true}>
+                {I18n.t("authentication.cie.temporaryBlockedCieContent4-bold")}
+              </Text>
+              <Text style={styles.text}>
+                {I18n.t("authentication.cie.temporaryBlockedCieContent5")}
+                <Text style={styles.text} bold={true}>
+                  {I18n.t(
+                    "authentication.cie.temporaryBlockedCieContent5-bold"
+                  )}
+                </Text>
+              </Text>
+            </Text>
+            <Text style={styles.text}>
+              {I18n.t("authentication.cie.temporaryBlockedCieContent6")}
+            </Text>
```

Comment:
I decided to use segments of the plain text as in the current SPID information screen because using the markdown the "Download/Open CieID" button is shown before the completion of the loading of the text in the Markdown component.

Reply:
You're right, it could be confusing for users. I suggest to use the <Markdown> component and update the component itself to display also once the loading is completed. 

Into `<Markdown>`:
add `footerProps?: FooterWithButtonsProps;` to the compoent props and, inside the `React.Fragment`: 
```
 {!isLoading && this.props.footerProps && 
          <FooterWithButtons
            {...this.props.footerProps}
          />
        }
```

Here how the render of `TemporaryBlockedCie` can be reduced:

```
public render(): {
    const footerProps: FooterWithButtonsProps = {
      type: "TwoButtonsInlineHalf",
      leftButton: {
        block: true,
        cancel: true,
        onPress: this.props.navigation.goBack,
        title: I18n.t("global.buttons.cancel")
      },
      rightButton: {
        block: true,
        primary: true,
        onPress: () => Alert.alert(I18n.t("global.notImplemented")),
        title: I18n.t("authentication.cie.temporaryBlockedCieDoneButton")
      }
    };

    return (
        <BaseScreenComponent goBack={true}>
          <H1>{I18n.t("authentication.cie.temporaryBlockedCieTitle")}</H1>
          <Markdown footerProps={footerProps}>{THE NEW MARKDOWN}</Markdown>
        </BaseScreenComponent>        
    );
  }
```